website/docs/reference/dbt_project.yml.md

```diff
@@ -96,6 +97,27 @@ vars:
 
 </File>
 
+## The `+` prefix
```

Comment:
This is already a [separate ref page](https://docs.getdbt.com/reference/resource-configs/plus-prefix). Every use of `+` in the docs links to this page. Will this section cause confusion or hurt the search?

Reply:
Good catch, might be better to link to that page instead!

Reply:
added a link to that page and left the section just to reiterate